examples/03-magnetics/plot_inv_mag_MVI_Sparse_TreeMesh.py

```diff
@@ -350,7 +350,6 @@
     minGNiter=1,
     beta_tol=0.5,
     coolingRate=1,
-    coolEps_q=True,
```

Comment:
If we plan to keep this example using `Update_IRLS`, we shouldn't need to delete this line (at least for now), right?

Reply:
True